nomt/src/lib.rs

```diff
@@ -469,6 +470,7 @@ impl<T: HashAlgorithm> Nomt<T> {
         session: &mut Session,
         actuals: Vec<(KeyPath, KeyReadWrite)>,
         witness: bool,
+        into_overlay: bool,
```

Comment:
Nit: would appreciate if the call sites had `/** into_overlay */`, because otherwise it's hard to derive the sense from the following line

```
self.update_inner(&mut session, actuals, true, false)?;
```

without having an IDE (like in PRs) or going to the definition.